josm-presets/de-signals-bostrab.xml

```diff
@@ -576,6 +576,38 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 			</item>
 		</group>
 		<group name="Electricity Signals (St)" de.name="Schaltsignale (St)">
+			<item name="Signalcontact (St 1)" de.name="Signalkontakt (St 1)" type="node">
```

Comment:
There is a SVG image on Wikimedia Commons. Could you please convert it into a PNG of 18x18 pixel size, save it as styles/icons/de/bostrab/st1-18.png and include it here?